source/endpoints/intro.rst

```diff
@@ -0,0 +1,73 @@
+Introduction
+============
+
+**BEdita 4** exposes a set of default endpoints to model and handle your data and to perform many other
+tyipical backend API operations. Throughout this documentation a description of each of these **core** endpoints is given.
+
+All API examples are conventionally described using only **HTTP** requests and responses i.e. methods, url, headers and bodies.
+There are intentionally no API client language specific examples, so a basic knowledge of REST API concepts and HTTP is necessary.
+
+.. _api-headers:
+
+Headers
+-------
+
+Since the primary exchange format is JSON-API its `content negotiation rules <http://jsonapi.org/format/#content-negotiation>`_
+are applied with some extensions.
+
+Every request must contain an ``Accept`` header with one of the following values:
+
+    * ``application/vnd.api+json`` JSON-API default content type
+    * ``application/json`` is also fine; considered a synomym for JSON-API, is treated the same way
+
+Under special circumstances, and only for GET requests, also ``text/html``, ``application/xhtml+xml``, ``application/xhtml``
+and ``text/xhtml`` may be valid provided that HTML accept is configured (see :ref:`configuration-accept`):
```

Comment:
"[...] may be valid, provided that HTML accept is configured" or "[...] may be valid, if HTML accept is configured"